### PR TITLE
Fix `ModelExperimental` crash for Draco-compressed models with tangents

### PR DESCRIPTION
**Pull request by j9liu**
This PR fixes a bug where loading the [Draco-compressed WaterBottle model](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/WaterBottle/glTF-Draco) would cause a crash. Summary:
- The computation of bitangents never accounted for quantized normals.
- It also never accounted for morph target / skinning effects on the normal and tangent.
- The bitangent will now be calculated after those stages are applied.

Here's the [local sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=dVRNc9owEP0rGp9Mh8imkI8GwjSFJD0k00zINBdfhLxgTWTJI8mQtJP/3rVkCKHkwnhXb9/uvrcD18o6shKwBkMuiII1mYAVdUl/+1ycRdzHE60cEwpMFnXJ30wRYgtdy/xSiZI5OCfO1NDN1FtnmKlMcc9baSuc0AqZW9YJMw6/mOrThdHlFJYGwMYN31Hva5+mp4PBSe9bt0kMBjQ9Tvun6YkPjzPVcAfmotob9yewXKjlvXC8eNBSesr27Y65gjr9gAimbNw7SzueMQ2/u7wL8QL5tWElPBrELrQp34ffpiyVmjMZcPp6W3MDKBBz2vjuWaS0cQUKFqI1WJdFodumX20k8mdRMquA22TKHEvudA7SJjfSLW41y8EkT6iw+aGdk5As5eP10dQwrnfTdInoLNphLhsW5A7eUstxNloZUaIjK7CU5fkHjRr41UsFiAC0Wnp/mhli7zYhTYfzZuBuiH0DlNaIl/MDChV7huzqFAcGsj2Q7iaBvm6/W84rKUVltcjp083sbLB9PmBVeOq0EIsW4WH20iZ862yE93NTg+O93uOKwgJ1Bah4USvujzX2iA5p924F5NjIMDReP1+6zxf47CSZWr6vvTm9D3vuXelR77izAwpTz3WtGs5ZVYABahBbW/Kl3dEv32owDEsPo240su5VwniD+C7KCu+ycTKmNHFQVhIPySbzmj+Do9zaUE3IKNktHeViRUR+ceAvgXDJrMWXRS3lTPyBLBqPEsT/Vyq1F+XXCoxkrw2s6I1vQ5JSOkowPFzptJZzZvaY/wE) I used, which is just the water bottle loaded in the scene.

@ptrgags can you review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @j9liu!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/draco-compression-tangent-fix/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

File: Specs/Scene/ModelExperimental/GeometryPipelineStageSpec.js [hunk 1]
```diff
@@ -1195,6 +1198,130 @@ describe(
       });
     });
 
+    // The tangents in this model aren't quantized, but they still should not
+    // force the model to crash.
```
Comment (ptrgags) [inline Specs/Scene/ModelExperimental/GeometryPipelineStageSpec.js:1202]:
```suggestion
    // cause the model to crash.
```

File: Specs/Scene/ModelExperimental/GeometryPipelineStageSpec.js [hunk 2]
```diff
@@ -68,6 +68,8 @@ describe(
       "./Data/Models/GltfLoader/BuildingsMetadata/glTF/buildings-metadata.gltf";
     const dracoMilkTruck =
       "./Data/Models/DracoCompression/CesiumMilkTruck/CesiumMilkTruck.gltf";
+    const dracoWaterBottle =
+      "./Data/Models/GltfLoader/WaterBottle/glTF-Draco/WaterBottle.gltf";
```
Comment (lilleyse) [inline Specs/Scene/ModelExperimental/GeometryPipelineStageSpec.js:72]: Is there a smaller model that we can use? This one has some pretty large textures.

It looks like the glTF exporter in Blender has an option to output tangents, so maybe you could export a simple box and then Draco compress it with gltf-pipeline.

**Review by ptrgags — COMMENTED** [review]
@j9liu, I agree, the bottle model has rather large textures.

I'm also noticing the normal map doesn't seem to look right. Compare these:

* [Model Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=dVRNb9swDP0rgk/OkMrOmn6sTYN1yZodWqxoivXiiyIzsVBZMiQ5aTr0v4+WnDTN0oshko+P5CNhrpV1ZClgBYZcEQUrMgIr6pL+8b44i7i3R1o5JhSYLOqSv5kixBa6lvm1EiVzcEGcqaGbqbfOZaYyxT1vpa1wQitkbllHzDh8MXVM50aXY1gYABs3fEe9r8c0Pev3T3vfuo2j36fpSXp8lp568yRTDXdgLqq9dn8By4Va3AvHiwctpadsY3fMFdTpB0QwZePeedrxjGn47vLOxQvkN4aV8GgQO9emfG9+67JUas5kwOmbbc4EUCDmtPHVs0hp4woULFgrsC6LQrVNvdpI5M+iZFoBt8mYOZbc6RykTSbSzW81y8EkT6iw+aGdk5As5OPN0dgwrnfddIHoLNphLhsW5A67pZZjb7QyosSNLMFSlucfNGrgfidN3dhvmDTtdcPL06GQRrxcHNCj2JN/V5U4MJDtOXQ3Dtzi9t1y/pRSVFaLnD5Npuf9bfjAYkKo00IsLgTPsJc25ltnI7Pvmxpsb32PwwkL1BWg4nmtuD/N2CM6pJ24lYtjIcNwzfr52n0+wGcHyNTifezNoX2Yc+8mj3onnR1Q6Hqma9VwTqsCDFCD2NqSL+2MfvhWg8sw9GXUjQbWrSUMN4jvoqzwCptNxpQmDspK4tnYZFbzZ3CUWxuyCRkku6mDXCyJyK8O/AAIl8xajMxrKafiFbJoOEgQ/1+q1F6U30swkq0bWNEb3gYnpXSQoHk402ktZ8zsMf8D)
image
* [ModelExperimental Sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=dVRNc9owEP0rGp9Mh8imkI8GwjSFJD0k00zINBdfhLxgTWTJI8mQtJP/3rVkCKHkwnhXb9/uvrcD18o6shKwBkMuiII1mYAVdUl/+1ycRdzHE60cEwpMFnXJ30wRYgtdy/xSiZI5OCfO1NDN1FtnmKlMcc9baSuc0AqZW9YJMw6/mOrThdHlFJYGwMYN31Hva5+mp4PBSe9bt0kMBjQ9Tvun6YkPjzPVcAfmotob9yewXKjlvXC8eNBSesr27Y65gjr9gAimbNw7SzueMQ2/u7wL8QL5tWElPBrELrQp34ffpiyVmjMZcPp6W3MDKBBz2vjuWaS0cQUKFqI1WJdFodumX20k8mdRMquA22TKHEvudA7SJjfSLW41y8EkT6iw+aGdk5As5eP10dQwrnfTdInoLNphLhsW5A7eUstxNloZUaIjK7CU5fkHjRr41UsFiAC0Wnp/mhli7zYhTYfzZuBuiH0DlNaIl/MDChV7huzqFAcGsj2Q7iaBvm6/W84rKUVltcjp083sbLB9PmBVeOq0EIsW4WH20iZ862yE93NTg+O93uOKwgJ1Bah4USvujzX2iA5p924F5NjIMDReP1+6zxf47CSZWr6vvTm9D3vuXelR77izAwpTz3WtGs5ZVYABahBbW/Kl3dEv32owDEsPo240su5VwniD+C7KCu+ycTKmNHFQVhIPySbzmj+Do9zaUE3IKNktHeViRUR+ceAvgXDJrMWXRS3lTPyBLBqPEsT/Vyq1F+XXCoxkrw2s6I1vQ5JSOkowPFzptJZzZvaY/wE)
image -- you get dark patches that don't seem quite right

**Comment by j9liu** [comment]
@ptrgags The dark patches appear for the uncompressed water bottle even on `main`, so I don't think it's related to this PR. I'll open an issue later.

**Comment by j9liu** [comment]
@ptrgags okay, I exported a model from blender with tangents and draco-compressed it. Here's a [local sandcastle](http://localhost:8080/Apps/Sandcastle/index.html#c=dVRNc9owEP0rGp/sDpFNA0lKSKYp+eihmWZCprn4IuQFayJLHkkG0k7+e9eSoYSSC6NdvX27em8x18o6shSwAkMuiIIVmYAVTUV/+VycR9zHE60cEwpMHvXIn1wRYkvdyOJKiYo5GBFnGujl6i05z1WuuOettRVOaIXMHeuEGYcnpo7p3OjqGhYGwMYt31H/8zHNTgeDk/6XXpsYDGg2zI5PsxMfDnPVcgfmst4b9zuwQqjFg3C8fNRSesru7p65kjr9iAimbNw/yxLPmIXfXd65WENxa1gFTwaxc22qf8NvU5ZKzZkMOH27rbkDFIg5bXz3PFLauBIFC9EKrMuj0G3TrzES+fMondbAbXrNHEvvdQESz4ZxPdFVbcBaFDH9ptfPwpVPTC1AObsf04V08zzaIa9aIqQP9lLLcTxaG1GhKUuwlBXFO5la+M26BkQgIZPeojtkjb3hhLQdRu3MvRD7BqiuEevRAZHKPU92pYoDA9nuSG+TQGu3547zRkpRWy0K+nw3PRtsrw+4Fa6SDmLRJdzNYRu9JRvp/djU4HSvD/hCYYG6ElQ8bxT36xp7REK6Z3f6cexjGFqvX67cx/N/tJStTduqzfK9e+benh71h8kOKEw9041qOad1CQaoQWxjySfSzzbIpJPgPDz6POpFY+teJVxuEF9FVeNmtkbGlKYOqlrif9ims4a/gKPc2lBNyDjdLR0XYklEcXHgo0C4ZNbizbyRcip+Qx5djlPE/1cqtRfl5xKMZK8trOxf/ghJSuk4xfBwpdNazpjZY/4L) to prove that it loads without crashing.

Let me know if anything else needs fixing!

**Comment by ptrgags** [comment]
Thanks @j9liu, looks good now!

**Pull request merged**